projects/RabbitMQ.Client/client/impl/ISession.cs

```diff
@@ -44,12 +44,14 @@
 
 namespace RabbitMQ.Client.Impl
 {
-    interface ISession
+    delegate void CommandReceivedAction(in IncomingCommand cmd);
```

Comment:
for consistency reasons should this have the `internal` keyword?

Reply:
yes, good catch